index.html

```diff
@@ -55,13 +63,19 @@ <h2 id="supportingtext">I'm a Software Developer</h2>
       </div>
       <div>
         <section class="recentworks">
+          <div class="zero">
```

Comment:
Also, your headline work section is not matching the Figma template. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177905737-8c087d88-bc17-446b-9f0f-c0243f8d48f6.png) | ![image](https://user-images.githubusercontent.com/33268927/177905835-87f6178b-7dd0-486a-86c3-5414bf059a84.png) |